.azure-pipelines/azure-pipelines.yml

```diff
@@ -9,33 +9,41 @@ trigger:
       - master
 
 variables:
+  GOPATH: $(Agent.BuildDirectory)/go
+  PATH: $(Agent.BuildDirectory)/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GOVER: 1.13.4
   branch: $[ coalesce(variables['system.PullRequest.TargetBranch'], variables['build.SourceBranchName']) ]
 
 pool:
   vmImage: ubuntu-16.04
-container:
-  image: awjh/fabric-contract-api-go-env:0.0.2
```

Comment:
why is this not needed anymore?

Reply:
I changed it so that azure installs everything on the ubuntu image it has rather than relying on a docker image with it pre-installed. This allows use of the default azure tools like the node tasks which simplifies the permissions on global install of npm modules